src/applications/personalization/profile/tests/e2e/address-validation/addressValidation.js

```diff
@@ -149,7 +149,7 @@ export const createAddressValidationResponse = type => {
             countryCodeIso3: 'USA',
             countyCode: '06001',
             countyName: 'Alameda',
-            stateCode: 'CA',
+            stateCode: 'MD',
```

Comment:
I needed to make the state match the user's starting address which is set in user-36:https://github.com/department-of-veterans-affairs/vets-website/blob/c57ee0a0437e7ea79c63072de14d84660ecce90e/src/applications/personalization/profile/tests/fixtures/users/user-36.json#L93